### PR TITLE
Add manifest for io.github.chcandido.brModelo

### PR DESCRIPTION
**Pull request by guesant**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**

The app is under GPL v3 license (https://github.com/chcandido/brModelo/blob/master/LICENSE), so we can distribute it freely.

I made an issue to distribute the program as Flatpak, but no response from the author so far (https://github.com/chcandido/brModelo/issues/21). (I think he's too busy).

- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.chcandido.brModelo.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  <id>io.github.chcandido.brModelo</id>
+  <launchable
+    type="desktop-id"
+  >io.github.chcandido.brModelo.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <developer_name>Carlos Henrique Candido</developer_name>
+  <name>brModelo</name>
+  <summary
+  >Entity-relationship diagram tool for making conceptual and logical database models.</summary>
```
Comment (hfiguiere) [inline io.github.chcandido.brModelo.metainfo.xml:12]:
```suggestion
  <summary>Entity-relationship diagram tool for making conceptual and logical database models.</summary>
```

**Comment by hfiguiere** [comment]
bot, build io.github.chcandido.brModelo

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.github.chcandido.brModelo.metainfo.xml (hunk 1 above)
Comment (guesant) [reply io.github.chcandido.brModelo.metainfo.xml]: fixed in [`fabed18` (#2876)](https://github.com/flathub/flathub/pull/2876/commits/fabed18e1de91483e74f8d6b6930eaed25d22bab).

File: io.github.chcandido.brModelo.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      - ln -s ${FLATPAK_DEST}/br-modelo/br-modelo ${FLATPAK_DEST}/bin/br-modelo
+
+      - install -Dm644 io.github.chcandido.brModelo.desktop ${FLATPAK_DEST}/share/applications/io.github.chcandido.brModelo.desktop
+      - install -Dm644 io.github.chcandido.brModelo.metainfo.xml ${FLATPAK_DEST}/share/metainfo/io.github.chcandido.brModelo.metainfo.xml
+      - install -Dm644 io.github.chcandido.brModelo.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/io.github.chcandido.brModelo.svg
+
+    sources:
+      - type: git
+        url: https://github.com/chcandido/brModelo
+        commit: 06bd0f416003636351f6e0bfb06cdfc1c74bd529
```
[the remark this answers is not in the document]
Comment (guesant) [reply io.github.chcandido.brModelo.yaml:76]: fixed in [`fabed18` (#2876)](https://github.com/flathub/flathub/pull/2876/commits/fabed18e1de91483e74f8d6b6930eaed25d22bab).

**Comment by hfiguiere** [comment]
bot, build io.github.chcandido.brModelo

**Comment by barthalion** [comment]
bot, build io.github.chcandido.brModelo

File: io.github.chcandido.brModelo.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+<component type="desktop-application">
+  <id>io.github.chcandido.brModelo</id>
+  <launchable type="desktop-id">io.github.chcandido.brModelo.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <developer_name>Carlos Henrique Candido</developer_name>
+  <name>brModelo</name>
+  <summary>Entity-relationship diagram tool for making conceptual and logical database models.</summary>
+  <description>
+    <p>brModelo is a entity-relationship diagram tool for making conceptual and logical database models.</p>
```
Comment (barthalion) [inline io.github.chcandido.brModelo.metainfo.xml:11]: It launches in Portuguese(?) for me, and I don't see an option to change it back to English. Unless I missed it, can you include that in the description?

Comment (yuuyins) [reply io.github.chcandido.brModelo.metainfo.xml:11]: @barthalion This software only has the Portuguese language. What do you mean by "that"?

Comment (hfiguiere) [reply io.github.chcandido.brModelo.metainfo.xml:11]: Include in the description that the application is only available in portuguese.

Comment (yuuyins) [reply io.github.chcandido.brModelo.metainfo.xml:11]: @guesant did you see this?

**Pull request closed without merging**